### PR TITLE
Merged fill/stroke shape functions together. Fixes #5

### LINKED ISSUE #5: Merge draw<Shape> and drawFilled<Shape>
Just pass additional parameters?

### PR DESCRIPTION
**Pull request by ojdon**

### CODE REVIEW

File: components/draw.js [hunk 1]
```diff
@@ -45,38 +43,27 @@ function ConvergameDraw() {
...
   };
 
-  this.filledRectangle = function(x, y, width, height, strokeStyle, fillStyle)
+  this.rectangle = function(x, y, width, height, strokeStyle, fillStyle)
   {
-    this.convergame.ctx.strokeStyle = strokeStyle;
-    this.convergame.ctx.fillStyle = fillStyle;
-    this.convergame.ctx.fillRect(this.prepNum(x*this.getXScale()), this.prepNum(y * this.getYScale()), this.prepNum(width*this.getXScale()), this.prepNum(height*this.getYScale()));
+    if(typeof(fillStyle) != 'undefined') {
+      console.log(fillStyle);
```
Comment (DivineOmega) [inline components/draw.js]: `console.log` left in here.

File: components/draw.js [hunk 2]
```diff
@@ -45,38 +43,27 @@ function ConvergameDraw() {
...
+    if(typeof(fillStyle) != 'undefined') {this.convergame.ctx.fill();}
   };
 
-  this.filledRectangle = function(x, y, width, height, strokeStyle, fillStyle)
+  this.rectangle = function(x, y, width, height, strokeStyle, fillStyle)
   {
-    this.convergame.ctx.strokeStyle = strokeStyle;
-    this.convergame.ctx.fillStyle = fillStyle;
-    this.convergame.ctx.fillRect(this.prepNum(x*this.getXScale()), this.prepNum(y * this.getYScale()), this.prepNum(width*this.getXScale()), this.prepNum(height*this.getYScale()));
+    if(typeof(fillStyle) != 'undefined') {
```
Comment (DivineOmega) [inline components/draw.js]: Instead of this if, perhaps just use `ctx.rect` followed by `ctx.stroke` and then an optional `ctx.fill`?

**Comment by ojdon** [comment]
I've updated the rectangle method to match how the circle rendering method works, based on your suggestion above.

**Comment by DivineOmega** [comment]
Looks good to me!

**Pull request merged**